SYCL/ESIMD/regression/Inputs/complex-lib-esimd.cpp

```diff
@@ -0,0 +1,26 @@
+#include <CL/sycl.hpp>
+#include <sycl/ext/intel/experimental/esimd.hpp>
+
+static constexpr const int VL = 4;
+
+sycl::event add(size_t n, sycl::buffer<double, 1> &buf_a,
```

Comment:
This function/kernel is defined, but not called. 'int main()' calls only 'iota'. Should 'add' also be called?

Reply:
I think so, even though it wasn't part of the original scenario. That would strengthen the test.

Reply:
Fixed in dc06d2d38ea1461354015c5f06e1cc9176dfcc20